packages/@guidesmiths/cuckoojs-schematics/src/systemic-scaffolding/files/makefile

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
what's this file for?

Reply:
It's a makefile intended for executing task in a simpler manner. I kept it because it's on the original systemic template and can be used during the CI.